### PR TITLE
JBPM-7466 Process Instances: Changing views renames process variables…

### PR DESCRIPTION
**Pull request by bxf12315**
… columns

(eder) child: https://github.com/kiegroup/jbpm-wb/pull/1224

### CODE REVIEW

**Comment by bxf12315** [comment]
@ederign updated it, Please review it again.

**Comment by cristianonicolai** [comment]
@bxf12315 looks like some tests are failing, please take a look.

File: uberfire-extensions/uberfire-widgets/uberfire-widgets-table/src/test/java/org/uberfire/ext/widgets/table/client/UberfireColumnPickerTest.java [hunk 1]
```diff
@@ -167,5 +167,37 @@ public void testAddColumnBeforeActions(){
...
+        columnPicker.addColumn(name);
+        columnPicker.addColumn(age);
+        columnPicker.addColumn(description);
+        columnPicker.addColumn(actions);
+        columnPicker.addColumn(etc);
+
+        assertEquals(4, columnPicker.getDataGrid().getColumnCount());
+        assertEquals("name", columnPicker.getDataGrid().getColumn(0).getDataStoreName());
+        assertEquals("age", columnPicker.getDataGrid().getColumn(1).getDataStoreName());
+        assertEquals("actions", columnPicker.getDataGrid().getColumn(3).getDataStoreName());
```
Comment (cristianonicolai) [inline uberfire-extensions/uberfire-widgets/uberfire-widgets-table/src/test/java/org/uberfire/ext/widgets/table/client/UberfireColumnPickerTest.java]: can you just order the assert in the order it is expected the columns to show? That would make it easier to read.

Comment (bxf12315) [reply uberfire-extensions/uberfire-widgets/uberfire-widgets-table/src/test/java/org/uberfire/ext/widgets/table/client/UberfireColumnPickerTest.java]: ok

**Comment by bxf12315** [comment]
@cristianonicolai updated

**Comment by ederign** [comment]
@bxf12315 what is the parent issue of this?

**Comment by bxf12315** [comment]
> @bxf12315 looks like some tests are failing, please take a look.





> @bxf12315 what is the parent issue of this?

Parent PR in here https://github.com/kiegroup/jbpm-wb/pull/1224.
Issue in here https://issues.jboss.org/browse/JBPM-7466

**Comment by cristianonicolai** [comment]
@ederign should be good to merge now, along with [kiegroup/jbpm-wb#1224](https://github.com/kiegroup/jbpm-wb/pull/1224)

**Pull request merged**